.github/workflows/v2-docs-api.yaml

```diff
@@ -188,9 +188,9 @@ jobs:
           path: ~/.m2/repository/io/stargate
           key: snapshots-${{ github.sha }}
 
-      - name: Build & Test
+      - name: Build & Test Stargate v2 Quarkus-based APIs
```

Comment:
We probably want to renaming the workflow file itself from `v2-docs-api.yaml`. But can be done as follow-up.